cross/zvbi/patches/09_fix-FTBFS-GCC6.patch

```diff
@@ -0,0 +1,34 @@
+Description: fix a FTBFS with GCC-6 (narrowing conversion)
```

Comment:
DSM-6.1 uses gcc4 and DSM7 uses gcc5.  Is this patch really necessary?  To keep as long as it has no negative impacts...

Reply:
at some point (from my experience) you need to bump up gcc since you have to do too many back-porting hacks (lesson learned building EL8 packages for RedSleeve (ARMv5). So keeping this until its applied upstream.

Reply:
Fully agree but we use the default Synology toolchain and until we are in a better situation this is all we have.

As such we are stuck with these old gcc versions until we figure how to build updated cross-compilers for all arches providing newer version of gcc (and that is a hell lot of work, some initial work was done here https://github.com/SynoCommunity/spksrc/pull/4469 and is heavily tied to the kernel sources which Synology hasn't even yet released for DSM7, which sucks really much).

In the meantime, if we do include updated gcc patches on the build it better not have any impact using the older gcc we have.